api/src/controllers/platform/config.ts

```diff
@@ -46,12 +45,12 @@ export async function platformConfig(
     response.contact = meta.contact
   }
   if (meta?.mail) {
-    response.mail = meta.mail
+    response.mail = { ...meta.mail, port: String(meta.mail.port) }
```

Comment:
Yeah, weird.

Reply:
I just figured this out. The port is indeed reported as a string because now it is being generated via the `schema.yaml` file. And I think that must be a mistake in `schema.yaml` line 605.